src/clj/harja/palvelin/palvelut/laadunseuranta.clj

```diff
@@ -184,7 +177,10 @@
                    toimenpideinstanssi vakiofraasi poistettu] :as sanktio} laatupoikkeama urakka]
   (log/debug "TALLENNA sanktio: " sanktio ", urakka: " urakka ", tyyppi: " tyyppi ", laatupoikkeamaan " laatupoikkeama)
   (when (id-olemassa? id) (vaadi-sanktio-kuuluu-urakkaan db urakka id))
-  (let [urakan-tiedot (first (urakat/hae-urakka db urakka))
+  (let [summa (if (decimal? summa)
+                (Double/parseDouble (str summa))            ;; Math/abs ei kestä BigDecimaalia, joten varmistetaan, ettei sitä käytetä
```

Comment:
Miksi tässä muutetaan summa stringiksi ja sen jälkeen parsitaan doublena?
Tämän pitäisi ajaa sama asia:
```suggestion
                (double summa)            ;; Math/abs ei kestä BigDecimaalia, joten varmistetaan, ettei sitä käytetä
```

Reply:
Paljon nätimpi näin. Kiitos!